composer.json

```
@@ -0,0 +1,12 @@
+{
+    "name": "mzur/kirby-calendar-plugin",
+    "description": "A plugin for the Kirby CMS to easily implement an event calendar.",
+    "type": "composer-plugin",
+    "license": "GNU GPL v2",
+    "autoload": {
+        "psr-4": {
+            "Mzur\\KirbyCalendarPlugin\\": "src/"
+        }
```

Thanks! This won't initialize the plugin correctly, though, as the `index.php` is no longer executed. Instead, a `files` autoload section should work:
```suggestion
        "files": [
           "index.php"
        ]
```
Also, the `index.php` will need a `use` statement for the `Kirby` class, similar to [this one](https://github.com/mzur/kirby-uniform/blob/b0854a60ccce50162ad34ea0bf8d21c84b648ab9/config.php#L3).

You could also add [`authors` and `support`](https://github.com/mzur/kirby-uniform/blob/b0854a60ccce50162ad34ea0bf8d21c84b648ab9/composer.json#L7-L17) sections.

Could you please apply the changes and test them?